data/tribes/initialization/atlanteans/starting_conditions/discovery.lua

```diff
@@ -25,6 +25,9 @@
 --                         starting condition, e.g. ``{ "scripting/win_conditions/hq_hunter.lua" }`` if the starting condition
 --                         is incompatible with the win condition "HQ Hunter".
 --
+--    **uses_map_starting_position**: *Optional* Whether this starting condition considers the player's
+--                                    starting position as defined by the map. Defaults to ``true``.
+--
```

Comment:
For me this description is hard to understand. There should be at least a hint to the checkbox `custom starting position`. E.g.:

"... If this set to false and all players are set to use a starting condition with no headquarter (e.g. "Discovery"), the Checkbox for _Custom starting position_ is disabled. If only one player is set to use a starting condition without headquarters, _Custom starting position_ is enabled, but for this player it has no effect. Defaults to: ``true``."